### PR TITLE
3.7.0 : Correcting login/logout lang strings

### PR DESCRIPTION
**Pull request by infograf768**
Since we now use modals to choose menu items redirections, the UI has changed.
screen shot 2016-10-07 at 10 23 46

This PR adapts the language strings to the new UI
Examples below for mod login. Same changes for login form menu item and logout menu item.
screen shot 2016-10-07 at 10 23 55

screen shot 2016-10-07 at 10 28 53

### CODE REVIEW

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 463bbb61864cfa2754e2c8af8611e737644784b7<br /><br />on code review. seems fine to me<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12337">issues.joomla.org/tracker/joomla-cms/12337</a>.</sub>

**Comment by zero-24** [comment]
Agree @brianteeman can you confirm on review?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12337">issues.joomla.org/tracker/joomla-cms/12337</a>.</sub>

File: language/en-GB/en-GB.mod_login.ini [hunk 1]
```diff
@@ -6,9 +6,9 @@
...
 MOD_LOGIN_FIELD_GREETING_LABEL="Show Greeting"
-MOD_LOGIN_FIELD_LOGIN_REDIRECTURL_DESC="Select the page the user will be redirected to after a successful login. Select from all the pages listed in the dropdown menu. Choosing &quot;Default&quot; will return to the same page."
+MOD_LOGIN_FIELD_LOGIN_REDIRECTURL_DESC="Select or create the page the user will be redirected to after a successful login. Select from all the pages listed in the dropdown menu. If none are selected the redirection will be to the same page."
 MOD_LOGIN_FIELD_LOGIN_REDIRECTURL_LABEL="Login Redirection Page"
-MOD_LOGIN_FIELD_LOGOUT_REDIRECTURL_DESC="Select the page the user will be redirected to after successfully ending their current session by logging out. Select from all the pages listed in the dropdown menu. Choosing &quot;Default&quot; will return to the same page."
+MOD_LOGIN_FIELD_LOGOUT_REDIRECTURL_DESC="Select or create the page the user will be redirected to after successfully ending their current session by logging out. Select from all the pages listed in the dropdown menu. If none are selected the redirection will be to the same page."
```
Comment (brianteeman) [inline language/en-GB/en-GB.mod_login.ini]: Do we need this sentence, it seem redundant to me.

Select from all the pages listed in the dropdown menu

**Comment by brianteeman** [comment]
Do we need this sentence, it seem redundant to me.

*"Select from all the pages listed in the dropdown menu"*

Also I don't think the last sentence is required either as its also
redundant. It's also technically wrong as there is no redirection. You
can't be redirected to where you were before.

*"If none are selected the redirection will be to the same page.".*

.

**Comment by infograf768** [comment]
I agree concerning "Select from all the pages listed in the dropdown menu." we can indeed get rid of it.

But not sure concerning the second one as it IS a redirection, even if it is to the same page. It is an information to the user of what will happen when login or logout. Which is exactly the purpose of a description.

**Comment by brianteeman** [comment]
How about.
The default is to remain on the same page.

**Comment by infograf768** [comment]
Done. Please test. @andrepereiradasilva @brianteeman

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 4a22418276b254780dc1ab3a5c27399e3311c863<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12337">issues.joomla.org/tracker/joomla-cms/12337</a>.</sub>

**Comment by infograf768** [comment]
We need one more test to confirm this as there were changes
@andrepereiradasilva

**Comment by zero-24** [comment]
@yvesh can you take a look into the drone problems?

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 4a22418276b254780dc1ab3a5c27399e3311c863<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12337">issues.joomla.org/tracker/joomla-cms/12337</a>.</sub>

**Comment by yvesh** [comment]
@infograf768 can you update your branch from joomla staging?

@zero-24 The fixes of highlighter.js are not in his branch. That's probably the reason drone is failing here.

**Comment by zero-24** [comment]
lets see if that fixes the problem. Thanks @yvesh

**Comment by infograf768** [comment]
RTC as last change was to correct conflicts<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12337">issues.joomla.org/tracker/joomla-cms/12337</a>.</sub>

**Pull request merged**